### DISCUSSION TITLE
Welcome to open-source-rover Discussions!

### DISCUSSION THREAD

**Discussion by ericjunkins** [nasa-jpl/open-source-rover#234] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by apollokit** [comment]
Hi there, I'm Kit and I've been helping with maintenance on OSR. Studied aerospace engineering, so rovers are up my alley.

**Comment by SConaway** [comment]
Hi Guys! I am the leader of a high school robotics club which was _trying_ to build OSR last school year. In general, I am interested in contributing to cool open source projects such as this one.

**Reply by ericjunkins** [reply]
@SConaway when you say emphasis on trying to build it, was it that there were technical hurdles that you ran into and couldn't get past them? Or was it because of a different factor? Ie. Pandemic, not enough time, etc.

**Reply by ericjunkins** [reply]
Also always happy to have contributors! If you're specifically interested in trying to contribute we have many tasks that need assistance! It can be daunting to know where/how to start, feel free to reach out if you're interested

**Reply by SConaway** [reply]
That’s a fair question! It was due to nothing on the project’s side at all. Mainly, the pandemic and scheduling 10 kids prevented the completion of OSR. My school is one you’ve visited a few times and you gave us a demo of your own OSR model and some other robotics equipment last year. It was really wonderful and I am sure all enjoyed it!

As for contributing, I’d love to help but have been swamped with school and work for the past few weeks. I’ll probably start with the LaTeX to Markdown migration if there’s anything left by the time I’m free!

**Comment by ericjunkins** [comment]
I'll introduce myself past the auto generated message that got put here. I'm Eric, I'm (now) a data scientist at JPL, when I started there a few years ago I was a robotics engineer. The open source rover was my internship project there that grew afterwards into what it is now with the help of many people both at jpl and in the open source community.

**Comment by ghost** [comment]
Hello everyone. I am Anthony and a big fan of robots and space, along with STEM.

Hoping to one day build this rover.

**Reply by Achllle** [reply]
Hope you build one too! 100% worth it :)

**Comment by Achllle** [comment]
Hi all! I'm Achille, a professional robotics engineer originally from Belgium but been living in the US (SF) for the last 5 years. I started building my rover in late 2019 and have contributed to the repositories quite a bit. I've written a few blogs about the rover ([1](https://www.freedomrobotics.ai/blog/building-the-nasa-jpl-open-source-rover), [2](https://www.freedomrobotics.ai/blog/how-to-pick-navigation-sensors-for-autonomous-mobile-robot), [3](https://www.freedomrobotics.ai/blog/tuning-odometry-for-wheeled-robots), and [4](https://www.freedomrobotics.ai/blog/deploying-on-mars-mapping-localization)), take a look for some inspiration! I'm also a maintainer of this project, feel free to message if you want to contribute!

**Comment by ChenZhiYongA** [comment]
Hi Guys, I am a developer from China and a fan of CNSA and NASA. I hope to make this car. If you too with the same hobbies, you can join me.

**Comment by dangerousthing** [comment]
Hi,
I'm Jay and I'm a retired programmer. I plan on building one of these, especially if it can handle normal stairs. I'm not as young as I once was, and don't think I can carry this up or down my apartment stairs.

**Reply by Achllle** [reply]
Welcome, Jay! It really depends what 'normal' stairs look like since there's no standard out there. The angle of the steps, depth, height, as well as their smoothness matter for the rover to be able to grip them and climb them consistently.
That being said, since the rover is made out of aluminum, it is relatively lightweight. I can weigh mine if you'd like.

Our [Slack group](https://join.slack.com/t/opensourcerover/shared_invite/zt-mfe0viua-FhcYsNb3S9vUra99uF3IBw) is a great forum to join if you'd like to join the community before deciding to build one.

**Comment by coding-yost** [comment]
Hello,
I am CSUN student due to graduate at the end of this semester. I saw this project at Scale21x and thought it'd be fun to give it a try. I have some embedded software experience and am excited to develop those skills with this project.

**Comment by nstolos99** [comment]
Hey,
    I am a CTE Computer Science teacher tasked to build a nonfunctioning manned Mars Rover for a Musem exhibit. I am looking for any design ideas. My initial thought was to use this project as a starting point. Any advice or links would be appreciated

**Reply by Achllle** [reply]
Welcome! I'm not sure if this would be the best place to start. All of the parts are COTS and don't naturally scale to a larger size and everything is functional. If that's not a requirement and you're trying to minimize cost while allowing a human to sit in/on it, I think the final product would look quite different.
I don't have any links for relevant projects here - there aren't (m)any real Mars Rovers for humans out there yet :)
